h/app.py

```diff
@@ -94,6 +121,13 @@ def includeme(config):
         },
     })
 
+    # After commit events
+    config.add_request_method(EventQueue, name='after_commit_events', reify=True)
```

Comment:
Can't think of anywhere you can put the list off the top of my head besides on the request. I think `request._after_commit_event_queue` (mark it as private) is fine.